src/controllers/user.controller.js

```diff
@@ -0,0 +1,67 @@
+const userService = require('./../services/user.service');
+
+const getUsers = (req, res) => {
+  const users = userService.getAllUsers();
+
+  res.send(users);
+};
+
+const getOneUser = (req, res) => {
+  const { id } = req.params;
+  const user = userService.getUserById(+id);
+
+  if (!user) {
+    return res.status(404).send('Error');
+  }
+  res.send(user);
+};
+
+const createUser = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    return res.sendStatus(400);
+  }
+
+  const user = userService.createUser(name);
+
+  res.status(201).send(user);
+};
+
+const updateUser = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+
+  const user = userService.getUserById(+id);
+
+  if (!user) {
+    return res.status(404).send('Error');
```

Comment:
Consider providing a more descriptive error message when a user is not found, such as 'User not found'. This will help in debugging and understanding the issue better.